example_pipeline/data_io.py

```diff
@@ -9,7 +9,12 @@
 import pandas
 
 from argparse import ArgumentParser
-from collections import Iterable
```

Comment:
Python v3.3 changes to `collections` addressed in 58924e0445dcca2c314d75e7467c5960750e3205